SpatialGDK/Documentation/content/whatsnew.md

```diff
@@ -0,0 +1,25 @@
+# What's New in Alpha?
+
+*If you’ve used the pre-alpha GDK you may be interested in these latest changes.*
+
+## Iterative improvements
+- Upgraded the engine fork to support Unreal Engine 4.20.3.
+- Upgraded to the new SpatialOS load balancer, providing fast multi-worker iteration.
+- Removed the incremental build time overhead of using the GDK by implementing dynamic type bindings, avoiding the need to generate certain interop code. You can find more info [here]({{urlRoot}}/content/dynamic-typebindings)
+- Schema generation for all classes is now the default.
+- Clients no longer automatically connect to a deployment, unless you pass it a commandline argument. For example: 
+```"%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor-Win64-Debug.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" 127.0.0.1```
```

Comment:
Not sure why this is not rendering correctly. Due to the %?

Reply:
Do the three backticks need to be on a different line to the text to top and tail it?